src/ol_superset/pythonpath/superset_config.py

```diff
@@ -35,7 +35,7 @@
 # --------------------------------
 APP_NAME = "MIT OL Business Intelligence"
 # Specify the App icon. Useful for white-labeling
-APP_ICON = "/static/assets/images/superset-logo-horiz.png"
+APP_ICON = "../static/assets/images/OL_sub-brand_lockup_one-line_rgb_black.png"
```

Comment:
We'll get a version of this that says "Data Platform" above "Open Learning" 

@quazi-h does it have to be a .png, or would a .svg work?

Reply:
Yes, I believe an .svg is an accepted icon format here